ssl/ssl_locl.h

```diff
@@ -551,6 +551,8 @@ struct ssl_session_st {
         /* Session lifetime hint in seconds */
         unsigned long tick_lifetime_hint;
         uint32_t tick_age_add;
+        unsigned char *tick_nonce;
+        size_t tick_nonce_len;
```

Comment:
I notice that this is stored as a separate pointer/length value, rather than as an ASN1_OCTET_STRING. Is that preferred? I ask because I'm adding something to the session also in PR #3802, and was thinking of storing it in the SSL_SESSION as an ASN1_OCTET_STRING